front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChartV2.tsx

```diff
@@ -1,339 +1,85 @@
-import type { SpeedSpaceChart, SpeedSpaceSettingsType } from 'reducers/osrdsimulation/types';
-import { useAppDispatch } from 'store';
-import { dateIsInRange, isoDateWithTimezoneToSec } from 'utils/date';
+import type { SimulationPowerRestrictionRange } from 'common/api/generatedEditoastApi';
 
-import ElectricalProfilesLegend from './ElectricalProfilesLegend';
-import { prepareSpeedSpaceDataV2 } from './prepareData';
-import SpeedSpaceSettings from './SpeedSpaceSettings';
-
-const CHART_ID = 'SpeedSpaceChart';
-const CHART_MIN_HEIGHT = 250;
-const SETTINGS_TO_AXIS = {
-  altitude: CHART_AXES.SPACE_HEIGHT,
-  curves: CHART_AXES.SPACE_RADIUS,
-  slopes: CHART_AXES.SPACE_GRADIENT,
-};
+import { formatData } from './helpers';
 
 export type SpeedSpaceChartV2Props = {
-  initialHeight: number;
-  onSetChartBaseHeight: (chartBaseHeight: number) => void;
   trainSimulation: SimulationResponseSuccess;
   selectedTrainPowerRestrictions: SimulationPowerRestrictionRange[];
   pathProperties: PathPropertiesFormatted;
-  trainRollingStock?: LightRollingStock;
-  sharedXScaleDomain?: PositionScaleDomain;
-  setSharedXScaleDomain?: React.Dispatch<React.SetStateAction<PositionScaleDomain>>;
-  departureTime: string;
+  heightOfSpeedSpaceChart: number;
+  setHeightOfSpeedSpaceChart: React.Dispatch<React.SetStateAction<number>>;
 };
 
-/**
- * A chart to see the evolution of speed of one train on its journey
- *
- * Features:
- * - One train only (current selected)
- * - Vertical line to the current position
- * - 2 marchs displayed: base and alternative
- *
- */
+const SPEEDSPACECHART_MIN_HEIGHT = 400;
+const SPEEDSPACECHART_BACKGROUND_COLOR = 'rgb(247, 246, 238)';
+const SPEEDSPACECHART_PADDING_BOTTOM = 22.5;
+
 const SpeedSpaceChartV2 = ({
-  initialHeight,
-  onSetChartBaseHeight,
   trainSimulation,
   selectedTrainPowerRestrictions,
   pathProperties,
-  trainRollingStock,
-  sharedXScaleDomain,
-  setSharedXScaleDomain,
-  departureTime,
+  heightOfSpeedSpaceChart,
+  setHeightOfSpeedSpaceChart,
 }: SpeedSpaceChartV2Props) => {
-  const simulationIsPlaying = useSelector(getIsPlaying);
-  const speedSpaceSettings = useSelector(getSpeedSpaceSettings);
-
-  const [chart, setChart] = useState<SpeedSpaceChart | undefined>(undefined);
-  const [chartBaseHeight, setChartBaseHeight] = useState(initialHeight);
-  const [chartHeight, setChartHeight] = useState(initialHeight);
-  const [isActive, setIsActive] = useState(false);
-  const [localSettings, setLocalSettings] = useState(speedSpaceSettings);
-  const [resetChart, setResetChart] = useState(false);
-  const [showSettings, setShowSettings] = useState(false);
-
-  const ref = useRef<HTMLDivElement>(null);
-
-  const dispatch = useAppDispatch();
   const { t } = useTranslation('simulation');
 
-  const toggleSetting = (settings: SpeedSpaceSettingsType) => {
-    dispatch(updateSpeedSpaceSettings(settings));
-  };
+  const [baseHeightOfSpeedSpaceChart, setBaseHeightOfSpeedSpaceChart] =
```

Comment:
If I understand correctly, this height is used to resize the height of the speedspacechart. Is it still a feature @thibautsailly wants ? 

If yes, you probably noticed it already but the y ticks for the speed don't scale very well. Maybe some of them should hide when there is overlapping.

Reply:
yes it's the vertical zoom with the bottom enlarger button, you can see him on sketch

about the ticks I don't see any problem, can you add a screenshot?

Reply:
Sure : 

![Capture d’écran 2024-07-23 à 17 32 20](https://github.com/user-attachments/assets/2d6944ef-8fee-47c8-b8bd-d6a51e874930)

Reply:
ok I see, those ticks will be enhanced in the part 2 of implementation to fix how major/minor lines should be displayed/set, and if we still base it on dozen or not, for now I can't raise the minimum height if you want

Reply:
Maybe it can be done later in your part2. It seems like the operational point don't meet the sketch so I presume you will do that in the part2 PR you mentioned as well (is this part clear for POs for the integration ?).

Reply:
yes it's known since the beginning of the PI, we hoped we could do that as a bonus but didn't have enough time